docs/gambit/mutation-verifier.md

```diff
@@ -192,7 +192,7 @@ At the moment, there are a few ways in which `certoraMutate` can fail.
 Here are some suggestions on how to troubleshoot when that happens. 
 We are actively working on mitigating them.
 
-- There are currently no official Gambit binaries for Linux arm machines. That means that Gambit will not be installed with `pip install certora-cli`, and must be {ref}`built from source <build-gambit-from-source>`.
+- There are currently no official Gambit binaries for Linux arm or Windows. That means that Gambit will not be installed with `pip install certora-cli`, and must be {ref}`built from source <build-gambit-from-source>`.
```

Comment:
@urikirsh i don't think this is quite right. We do have binary for linux. @ozcodes led that effort: you can see the code here: https://github.com/Certora/gambit/blob/a97670e0bfe96b6f38bfe002181916a1754283b7/.github/workflows/gambit.yml#L10